scripts/buildRpm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
```git rev-parse --abbrev-ref HEAD```

or 

```git status | head -1```

The only issue I see with this is when we are in headless state which seems to be the way that we are doing it on jenkins when running master.  That should be easy to fix though if we ever decide to be on master for this repo (which I doubt)